Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_TOOL = xcodebuild
-BUILD_SCHEME = SQLite Mac
```

Comment:
I think it would be useful to leave this default in so that (fast) tests can be run from the command line with a simple `make` or `make test`, instead of a lengthy `make test BUILD_SCHEME='SQLite Mac'`.